src/graphql/resolvers/getOffersResolver.js

```diff
@@ -4,10 +4,25 @@ import removeEmptyStrings from "../utils/removeEmptyStrings";
 import GetCompanyResolver from "./getCompanyResolver";
 
 // Gets all offers
-const GetOffersResolver = async db => {
-  let offers = await GetMany(db, TABLES.Offer);
-  let res = [];
+const GetOffersResolver = async (db, studentId = "") => {
+  let offers;
+  let offerParams = { TableName: TABLES.Offer };
+  if (studentId) {
+    offers = await db
+      .query({
+        ...offerParams,
+        KeyConditionExpression: "student_id = :id",
+        ExpressionAttributeValues: {
+          ":id": studentId
+        }
+      })
+      .promise();
+  } else {
+    offers = await db.scan(offerParams).promise();
+  }
+  offers = offers.Items || [];
```

Comment:
I could be wrong, but I think Bo was working on a filtering service method that could do this whole operation in like a single line of code. @bbleyl , is that true, or was it something else you were working on?

Reply:
Either way, this seems to work great, so I am ok with merging it for now. We just may need to refactor it later.